cmd/publishing-bot/main.go

```diff
@@ -94,9 +96,12 @@ func main() {
 	if *rulesFile != "" {
 		cfg.RulesFile = *rulesFile
 	}
-	if *basePublishScriptPath != "" {
+	if *basePublishScriptPath != defaultPublishScriptPath {
 		cfg.BasePublishScriptPath = *basePublishScriptPath
+	} else if cfg.BasePublishScriptPath == "" {
+		cfg.BasePublishScriptPath = defaultPublishScriptPath
```

Comment:
Since command line flags are meant to override config values, we can simplify this:

```go
	if cfg.BasePublishScriptPath == "" {
		cfg.BasePublishScriptPath = *basePublishScriptPath
	}
```

Reply:
It does not cover the case that custom config value + custom flag value.